Description: Show WiFi Hotspot password in a monospace font
   * panels/network/network-wifi.ui
     - Add the Monospace font attribute to the security key label
Author: Will Cooke <will@whizzy.org>
Origin: ubuntu
Bug-Ubuntu: https://bugs.launchpad.net/gnome-control-center/+bug/1706421
Forwarded: https://bugzilla.gnome.org/show_bug.cgi?id=785413
Last-Update: 2017-07-26

--- gnome-control-center-3.24.3.orig/panels/network/network-wifi.ui
+++ gnome-control-center-3.24.3/panels/network/network-wifi.ui
@@ -2010,6 +2010,9 @@ here</property>
                     <property name="xalign">0</property>
                     <property name="label"></property>
                     <property name="selectable">True</property>
+                    <attributes>
+                      <attribute name="font-desc" value="Monospace"/>
+                    </attributes>
                   </object>
                   <packing>
                     <property name="left_attach">1</property>
